Dudedev 28117 add escaping information to usage text for security utility (#29842)
OpenLiberty/open-liberty: README.md

2. Run a gradle build.

>    Pre-requisite - Environment variable `JAVA_21_HOME` must be set.

Reviewer: ```suggestion
   Prerequisite - Environment variable `JAVA_HOME` must point to a Java 17 or Java 21 SDK.  If setting `JAVA_HOME` to  Java 17, you will also need to set `JAVA_21_HOME` to a Java 21 SDK.
```

Author: Updated as suggested.

        cd open-liberty/dev
        ./gradlew cnf:initialize
        ./gradlew assemble

>     Troubleshoot - If `./gradlew assemble` shows Out of Memory error then you could increase the memory in `dev/gradle.properties` file. Stop gradle daemons after you change the memory using `./gradlew --stop`. If it does not help, then try cleaning up maven repository (using `rm -rf ~/.m2`) and gradle caches (using `rm -rf ~/.gradle/caches`).

Reviewer: ```suggestion
    Troubleshooting - If `./gradlew assemble` shows out of memory error, you can increase the memory in `dev/gradle.properties` file. Stop the gradle daemon after you change the memory setting by using `./gradlew --stop`. If it does not help the problem, try cleaning up your maven repository cache (using `rm -rf ~/.m2`) and gradle caches (using `rm -rf ~/.gradle/caches`).
```

Author: Updated as suggested.
